README.md

```diff
@@ -57,7 +57,11 @@ Customizable rules should be enabled through [configuration](#action-configurati
 ### Action configuration <a name="action-configuration"></a>
 
 The configuration file should be placed in `.github/pr-custom-review.yml`
-(related to [built-in checks](#built-in-checks)).
+
+The configuration file is always read from the repository's default branch. For this reason it's recommended to commit the configuration file **before** the action's workflow file is added, otherwise the action will fail with `RequestError [HttpError]: Not Found` because the configuration does not yet exist in the default branch.
+```
+ERROR:  RequestError [HttpError]: Not Found
+```
```

Comment:
Oops my bad, I forgot to delete this block as part of my suggestion.

This block should not be needed because the error is already mentioned in the explanation above.

After that, I think it's good to go!

Reply:
Thanks. Commited

Reply:
Thanks. Fixed.